### PR TITLE
PHRAS-3179 : worker - Controler for other job like subtitling 

### PR DESCRIPTION
**Pull request by aynsix**
### Adds
  - PHRAS-3179: worker - controler for workerpopulate et workerjob
  - add route   :
    /admin/worker-manager/info-populate
    /admin/worker-manager/info-subtitle

### CODE REVIEW

File: lib/Alchemy/Phrasea/WorkerManager/Controller/AdminConfigurationController.php [hunk 1]
```diff
@@ -92,6 +95,82 @@ public function infoAction(PhraseaApplication $app, Request $request)
...
+        $workerPopulateFinished = $repoWorkerPopulate->findBy(['status' => WorkerRunningPopulate::FINISHED]);
+
+        $populateRunning = [];
+        $populateFinished = [];
+
+        /** @var WorkerRunningPopulate $running */
+        foreach ($workerPopulateRunning as $running) {
+            $data['databox_id'] = $running->getDataboxId();
+            $data['status']     = $running->getStatus();
+            $data['published']  = $running->getPublished();
```
Comment (jygaulier) [inline lib/Alchemy/Phrasea/WorkerManager/Controller/AdminConfigurationController.php:152]: WHY for "job" we have "created; started; finished", and for "populate" we have "published; created; finished" ???
We should try to implement "populate" as a job (see conversation) and refacto everything.

File: lib/Alchemy/Phrasea/WorkerManager/Controller/AdminConfigurationController.php [hunk 2]
```diff
@@ -92,6 +95,82 @@ public function infoAction(PhraseaApplication $app, Request $request)
...
+        $workerSubtitleRunning  = $repoWorkerJob->findBy(['type' => MessagePublisher::SUBTITLE_TYPE, 'status' => WorkerJob::RUNNING]);
+        $workerSubtitleFinished = $repoWorkerJob->findBy(['type' => MessagePublisher::SUBTITLE_TYPE, 'status' => WorkerJob::FINISHED]);
+
+        $subtitlesRunning = [];
+        $subtitlesFinished = [];
+
+        /** @var WorkerJob $running */
+        foreach ($workerSubtitleRunning as $running) {
+            $data = $running->getData();
+            $data['status']   = $running->getStatus();
```
Comment (jygaulier) [inline lib/Alchemy/Phrasea/WorkerManager/Controller/AdminConfigurationController.php:112]: here it's always "running"...
btw, if we **really** want to return 2 separate arrays "running" and "finished", we must be more generic :
repo->findall(), then fill arrays by type --> smaller code, no code change for new type.

**Comment by jygaulier** [comment]
tooooo much duplicated code / routes.
We must have 1 route ".../{{job_type}}/progress".
A job must implement a "getProgress()" method.
Ideally when adding a new job type, we should only declare the type and implement the worker. Everything else should be automatic.
We also should try to transform the worker-populate to a job "populate"

**Comment by nmaillat** [comment]
PHRAS-3179 close in favor of PHRAS-3170 pr #3574

**Pull request closed without merging**